Model/SalesOrder/SalesOrdersPdf.php

```diff
@@ -59,10 +69,14 @@ public function setSalesOrders(array $salesOrders)
      */
     public function build()
     {
+
         foreach ($this->salesOrders as $salesOrder) {
+			$user = $salesOrder->getUser();
+			$association = $salesOrder->getBranchOccurrence()->getBranch()->getAssociation();
+			$balance = $this->subscriptionRepository->getBalanceUserAndAssociation($user,$association);
```

Comment:
Ne pas instancier de variable pour rien.
$this->subscriptionRepository->getBalanceUserAndAssociation(
$salesOrder->getUser(),
$salesOrder->getBranchOccurrence()->getBranch()->getAssociation()
);

Il aurait été préférable que la méthode ramène la subscription directement et non la balance. La méthode peut être appelée ailleur.